ansible/idr-proxy.yml

```diff
@@ -49,14 +49,22 @@
         }}:8080
     when: "{{ groups[idr_environment | default('idr') + '-a-dockermanager-hosts'] is defined }}"
 
-  - name: Configure rsync port in selinux
+  - name: Install packages for selinux
```

Comment:
Can you use https://github.com/openmicroscopy/ansible-role-selinux-utils#selinux-utils instead and make any se* tasks conditional on `selinux_enabled`.